views/posts/new.hbs

```diff
@@ -1,5 +1,5 @@
-<form id="new-post-form" action="/posts" method="post">
```

Comment:
Can probably just get rid of new post view and reference to view in routes/controller to keep codebase clean